packages/bson-transpilers/test/yaml/native.yaml

```diff
@@ -786,6 +821,7 @@ tests:
           java: 2d
           csharp: (int) 2
           object: Number(2)
+          php: '2'
```

Comment:
This is the "NumberSymbolArgsTemplate" template and it is designed like more recent converter for GO language:
````
    NumberSymbolArgsTemplate: &NumberSymbolArgsTemplate !!js/function >
        (lhs, arg, type) => {
            arg = arg === undefined ? 0 : arg;

            switch(type) {
                case '_string':
                    if (arg.indexOf('.') !== -1) {
                        return `(float) ${arg}`
                    }
                    return `(int) ${arg}`
                default:
                    return `${arg}`
            }
        }
````
If we have some string in the input we make some type casting. Otherwise we considering that we've got some number already in the input, so there is no need to make some typecasting.

Reply:
OK to resolve.